### ISSUE TITLE
B.4.8.0200. - Text Validation

### ISSUE THREAD

**Issue by aldefouw** [4bbakers/redcap_rsvc#76]
Feature: [B.4.8.0200. - Text Validation](https://github.com/4bbakers/redcap_rsvc/tree/v14.7.0/./Feature%20Tests/B/Field%20Validation_8/B.4.8.0200.%20-%20Text%20Validation.feature)

**Comment by mmcev106** [comment]
@ellisthomas, are you still actively working on this one?

**Comment by ellisthomas** [comment]
@mmcev106 Yes I am still actively working on this one. This one is failing due to steps for Time validated fields. _"When I clear field and enter "TEST" into the data entry form field labeled "Time HH:MM"_  If we replace this with a number that does not meet the validation it will run and give you the proper dialog box for the validation error.

Also, I believe the email variable is missing from the "Data Types" instrument. I do see it in the Codebook but it currently resides on the "Text Validation" instrument.

**Comment by ellisthomas** [comment]
@mmcev106 The following lines of code are looking to validate the Email field however it does not live on the Data Types instrument. Please advise on how you would like to proceed. 
```
# And I enter "TEST" into the data entry form field labeled "Email"
# And I should see a dialog containing the following text: "This field must be a valid email address (like joe@user.com). Please re-enter it now."
# And I click on the button labeled "Close" in the dialog box
# When I clear field and enter "" into the data entry form field labeled "Email"
```
